db/versions/0018.yml

```diff
@@ -0,0 +1,110 @@
+version: 18
+migrationScript: 0018-migration.sql
+downgradeScript: 0018-downgrade.sql
+methods:
```

Comment:
It looks like this leaves the existing entities functions in place, using the now-dropped entities table.  What is the rationale?  If we need to roll back the service deployment, must we also roll back the DB?

Reply:
Given that it's okay to lose data, I suppose we can rewrite them to return an empty set.

Reply:
That works.  On the other hand, you've already written the migration to copy the data over..